### ISSUE TITLE
How to adjust yaw rate on fly camera?

### ISSUE THREAD

**Issue by dtforrest** [vispy/vispy#1963]
I am using the fly camera in a 3D scene, its really nice that you have that built in. I was able to adjust the linear speed of the camera motion using view.camera.scale_factor=*some distance*, but even looking at the source code a bit it is not obvious to me how the base value for the pitch/yaw rate is determined. For what I am doing the yaw rate using the IJKL keys is rather extreme, I would like to reduce it significantly if possible so I can smoothly turn by holding the keys instead of getting whipped in circles.  Is there a value  similar to view.camera.scale_factor for the pitch/yaw?

Thanks

**Comment by djhoese** [comment]
I don't think this feature exists. I'm sorry I won't be of too much help as I've never really used the Fly camera, but I'm wondering if you added a "scale factor" to the yaw and pitch lines here:

https://github.com/vispy/vispy/blob/7f3f6f60c8462bb8a3a8fa03344a2e6990b86eb2/vispy/scene/cameras/fly.py#L209-L210

If that would have the same effect. I'd be open to a pull request if you could get something working. It would be greatly appreciated.

**Comment by kmuehlbauer** [comment]
@dtforrest Just checking in. Did you got any solution to your problem? It would be interesting you solved it.
